content/docs/tasks/traffic-management/request-routing/index.md

```diff
@@ -20,14 +20,39 @@ This task shows you how to configure dynamic request routing based on weights an
 
 ## Content-based routing
 
+The Bookinfo sample deploys multiple versions of each microservice, so we will start by creating DestinationRules
+that define the service subsets corresponding to each version, and the load balancing policy for each subset.
+
+1.
+
+    ```command
+    $ istioctl create -f @samples/bookinfo/routing/destination-rule-all.yaml@
+    ```
+
+    If you enabled `mTLS`, please run the following instead
+
+    ```command
+    $ istioctl create -f @samples/bookinfo/routing/destination-rule-all.yaml@
+    ```
+
+    > In a Kubernetes deployment of Istio, you can replace `istioctl`
+    > with `kubectl` in the above, and for all other CLI commands.
+    > Note, however, that `kubectl` currently does not provide input validation.
+
+    You can display the DestinationRules that are defined with the following command:
+
+    ```command-output-as-yaml
```

Comment:
Please use the modern {{< text bash yaml >}} sequence here instead of the backticks.
Also, don't indent the content, it should line up with the {{< text >}}